modules/optimizing-the-vllm-runtime.adoc

```diff
@@ -20,23 +20,20 @@ endif::[]
+endif::[]
+ifndef::upstream[]
+* If you used the pre-installed *vLLM ServingRuntime for KServe* runtime, you duplicated the runtime to create a custom version. For more information about duplicating the pre-installed vLLM runtime, see {rhoaidocshome}{default-format-url}/serving_models/serving-large-models_serving-large-models#adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models[Adding a custom model-serving runtime for the single-model serving platform].
+endif::[]
 * If you are using the vLLM model-serving runtime for speculative decoding with a draft model, you have stored the original model and the speculative model in the same folder within your S3-compatible object storage.
 
 
 .Procedure
 . From the {productname-short} dashboard, click *Settings* > *Serving runtimes*.
 +
 The *Serving runtimes* page opens and shows the model-serving runtimes that are already installed and enabled.
-+
-. Based on the runtime that you used to deploy your model, perform one of the following actions:
-+
-ifdef::upstream[]
-* If you used the pre-installed *vLLM ServingRuntime for KServe* runtime, duplicate the runtime to create a custom version and then follow the remainder of this procedure. For more information about duplicating the pre-installed vLLM runtime, see {odhdocshome}{default-format-url}/serving_models/serving-large-models_serving-large-models#adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models[Adding a custom model-serving runtime for the single-model serving platform].
-endif::[]
-ifndef::upstream[]
-* If you used the pre-installed *vLLM ServingRuntime for KServe* runtime, duplicate the runtime to create a custom version and then follow the remainder of this procedure. For more information about duplicating the pre-installed vLLM runtime, see {rhoaidocshome}{default-format-url}/serving_models/serving-large-models_serving-large-models#adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models[Adding a custom model-serving runtime for the single-model serving platform].
-endif::[]
-* If you were already using a custom vLLM runtime, click the action menu (&#8942;) next to the runtime and select *Edit*.
+. Find the custom vLLM model-serving runtime that you created, click the action menu (&#8942;) next to the runtime and select *Edit*.
 +
 The embedded YAML editor opens and shows the contents of the custom model-serving runtime.
 . To configure the vLLM model-serving runtime for speculative decoding by matching n-grams in the prompt, add the following arguments:
```

Comment:
Change so there is more than one substep:
. Configure ....in the prompt:
..  Add the following arguments:
.
.
.
.. Replace